Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,62 @@
+joint operational direction of the Node.js Technical Steering Committee (TSC)
+and Node.js Foundation Community Committee (CommComm).
+
+## Organization Roles
+
+### Owners
+
+The Node.js TSC Director, TSC Facilitator, and the two elected individual
+member representatives shall be granted Owner permissions within the Node.js
+GitHub Organization.
+
+Additional TSC, CommComm, and Core Technical Committee (CTC) members may be
+granted Owner permissions by request. Assuming there are no objections from any
+current voting TSC or CommComm member, such requests are approved automatically
+after 72 hours. If any objection is made, the request may be moved to a single
+vote in which a simple majority of both the TSC and CommComm is required to
+approve. Such requests must be posted as issues to either the nodejs/TSC or
+nodejs/community-committee repositories.
+
+Upon the completion of the TSC Director, TSC Facilitator, or elected individual
+member representative's terms, their Owner permissions within the GitHub
+organization shall be maintained *unless* the individual is also no longer a
+voting member of either the TSC or Community Committee.
+
+An individuals Owner permissions shall be automatically removed if they are no
+longer voting members of the TSC, CommComm, or CTC. Further, Owner permissions
+for any individual may be removed through single simple majority vote of both
+the TSC and CommComm.
+
+### Members
+
+GitHub users are added as members to the Node.js GitHub Organization when they
+are added to any Working Group. Organization Owners should add new members
+to the organization when requested by a Working Group.
+
+## Repositories
+
+Any organization member may request the creation of a new repository within the
+Node.js Foundation GitHub Organization. Provided there are no objections from
+any voting TSC or CommComm member, such requests are approved automatically
+after 72 hours. If any objection is made, the request may be removed to a single
+vote in which a simple majority of both the TSC and CommComm is required to
+approve. Such requests must be posted as issues to either the nodejs/TSC or
+nodejs/community-committee repositories.
+
+Any repository created under the Node.js GitHub Organization is considered to be
+a project under the ownership of the Node.js Foundation, and thereby subject
+to the Intellectual Property and Governance policies of the Foundation.
+
+No repository may be deleted, transfered in to, or transfered out of, the
+Node.js Foundation GitHub Organization without a simple majority vote of both
+the TSC and CommComm. In certain cases, Node.js Foundation Board of Directors
+approval may also be required.
+
+All repositories must have a README that clearly identifies the purpose of the
+repository, governance documentation that details how the repository is managed,
+and an indication of whether the repository falls under the operational
+direction of either the TSC or CommComm. 
```

Comment:
@nebrius iiuc CTC activities are just delegated from the TSC, from an outside or board view all of that falls under TSC responsibility, so I think the phrasing here is correct as it is